apps/web/src/components/Publication/FullPublication.tsx

```diff
@@ -32,11 +33,24 @@ const FullPublication: FC<FullPublicationProps> = ({ publication }) => {
     ? publication?.mirrorOf?.stats?.totalAmountOfCollects
     : publication?.stats?.totalAmountOfCollects;
   const showStats = mirrorCount > 0 || reactionCount > 0 || collectCount > 0;
+  const publicationRef = useRef<HTMLDivElement>(null);
+
+  const { pathname } = useRouter();
+  useEffect(() => {
+    if (
+      publicationRef?.current &&
+      publication.__typename === 'Comment' &&
```

Comment:
Same logic as in https://github.com/heyxyz/hey/blob/main/apps/web/src/components/Publication/Type/index.tsx#L30
not sure if there is a more elegant way, to avoid duplicated logic